### PR TITLE
Remove deprecated multiline dependency

### PR DESCRIPTION
**Pull request by fwilhe**
The dependency had some use pre es 2015 but is not required since then
and the author has deprecated it 4 years ago.

See issue https://github.com/SAP/grunt-openui5/issues/234

### CODE REVIEW

**Comment by matz3** [comment]
Thanks. Don't worry about the coveralls status. The slight decrease is not an issue.

**Comment by fwilhe** [comment]
Yes, I just wanted to know where that "new line" comes from 😬 

Just FYI I'm not that fond of (frontend) js/Grund, so I hope I did not unintentionally break anything. The change looked simple enough to me, but there might be things to watch out that I did not know about.

File: tasks/connect.js [hunk 1]
```diff
@@ -20,10 +19,10 @@ LiveReloadPluginLessCss.prototype.reload = function(path, options) {
 	// do only run this if less is not loaded (LiveReload LessPlugin will handle this)
 	if (!this.window.less) {
 		// reload stylesheets (css) also if a less file was changed
-		if (path.match(/\.less$/i)) {
```
Comment (matz3) [inline tasks/connect.js:23]: The backslash must be escaped by another backslash, not removed

File: tasks/connect.js [hunk 2]
```diff
@@ -20,10 +19,10 @@ LiveReloadPluginLessCss.prototype.reload = function(path, options) {
...
+		if (path.match(/.less$/i)) {
 			return this.window.LiveReload.reloader.reloadStylesheet(path);
 		}
-		if (options.originalPath.match(/\.less$/i)) {
```
Comment (matz3) [inline tasks/connect.js:26]: The backslash must be escaped by another backslash, not removed

File: package-lock.json [hunk 3]
```diff
@@ -2917,14 +2917,6 @@
 			"resolved": "https://registry.npmjs.org/ms/-/ms-2.1.3.tgz",
 			"integrity": "sha512-6FlzubTLZG3J2a/NVCAleEhjzq5oxgHyaCU9yYXvcLsvoVaHJq/s5xXI6/XXP6tz7R9xAOtHnSO/tXtF3WRTlA=="
 		},
-		"multiline": {
-			"version": "2.0.0",
-			"resolved": "https://registry.npmjs.org/multiline/-/multiline-2.0.0.tgz",
-			"integrity": "sha512-+HpXaUcV8PIGNNmuhtlaVmw4NH0W30/A5WP+rq6pxZYBjDslX/sXkFgL3Mgk1cSGGIICjWu4gNStkJXL6ZM2DQ==",
-			"requires": {
-				"strip-indent": "^2.0.0"
```
Comment (matz3) [inline package-lock.json:2925]: I wonder why strip-indent is not removed as it is not used by some other package.
Could you please remove the "node_modules" folder and run "npm install" again (with npm v6.x)?

**Review by matz3 — CHANGES_REQUESTED** [review]

**Comment by fwilhe** [comment]
@matz3 Thanks for your review. Regarding the regex escape, I think I tried this and saw an error when running the tests, but I did not investigate that further. Not sure about the test coverage for this kind of change, on my machines the test worked in both cases, but I don't know the context of this code so I can't judge correctness.

**Pull request merged**